mm2src/coins/utxo/utxo_builder/utxo_arc_builder.rs

```diff
@@ -219,3 +246,23 @@ where
         }
     }
 }
+
+pub trait BlockHeaderUtxoArcOps<T>: UtxoCoinBuilderCommonOps
+where
+    T: AsRef<UtxoCoinFields> + UtxoCommonOps + Send + Sync + 'static,
+{
+    fn spawn_block_header_utxo_loop_if_required<F>(
```

Comment:
Please add the `AbortHandle` similar to https://github.com/KomodoPlatform/atomicDEX-API/blob/db95a0e345b7bc8b5686ca56553f57a9749f2c13/mm2src/coins_activation/src/platform_coin_with_tokens.rs#L166

It's better to be stopped immediately on `stop` RPC call.